packages/sdk-codegen/src/sdkModels.ts

```diff
@@ -2364,6 +2391,13 @@ export class ApiModel implements ISymbolTable, IApiModel {
     return result
   }
 
+  /**
+   * Read-only accessor for private enum collection used within this source file as a "friend"
```

Comment:
This is actually protected but maybe I'm misreading the comment. From a purest standpoint as this is an object this is technically updatable by the caller. 
`return {...this,enumTypes}`

Reply:
it needs to be updateable